docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -109,10 +109,31 @@ besu operator generate-blockchain-config --config-file=<FILE> --to=<DIRECTORY> [
 besu operator generate-blockchain-config --config-file=config.json --to=myNetworkFiles
 ```
 
-The configuration file has 2 subnested JSON nodes. The first is the `genesis` property defining 
-the [IBFT 2.0 genesis file](../../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) except for the `extraData` string. The 
+The configuration file has 2 subnested JSON nodes. The first is the `genesis` property defining
+the [IBFT 2.0 genesis file](../../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) except for the `extraData` string. The
 second is the `blockchain` property defining the number of key pairs to generate.  
 
+### generate-log-bloom-cache
+
+Generates cached log bloom indexes for blocks. APIs use the cached indexes for improved log
+query performance.
+
+!!! note
+    Each index file contains 100000 blocks. The last fragment of blocks less that 100000 are not indexed.
+
+Must be manually executed to add new blocks to the indexes.
+
+The [`admin_generateLogBloomCache`](../API-Methods.md#admin_generateLogBloomCache) API can also be used to generate cached log
```

Comment:
The deep link (`admin_generateLogBloomCache`) isn't working - needs to be all lower-case `admin_generatelogbloomcache`

Reply:
Fixed.